_episodes/03-computing-pi.md

```diff
@@ -1,25 +1,24 @@
 ---
-title: "Computing Pi"
+title: "Understanding parallelization in python"
 teaching: 60
 exercises: 30
 questions:
 - "What is the GIL?"
 - "How do I parallelize an elementary program?"
-- "What is vector-based parallelism?"
-- "What is task-based parallelism?"
+- "What is data parallelism?"
+- "What is task parallelism?"
 - "How do I use multiple threads in Python?"
 objectives:
+- "Know how to rewrite a program in a vectorized form."
+- "Understand the difference between data and task-based parallel programming."
 - "Understand the GIL"
 - "Apply `numba.jit` to lift the GIL"
-- "Understand the difference between vectorized and task-based parallel programming."
 - "Recognize the primitive components of the queue/worker based model of execution."
 keypoints:
 - "Vectorized algorithms are both a blessing and a curse."
 - "If we want the most efficient parallelism on a single machine, we need to unlock the GIL."
 - "Numba helps you both speeding up and lifting code from the GIL."
 ---
-FIXME: Vector-based parallelization is explained well, but task-based is
-actually missing from this chapter.
 
 FIXME: Maybe it makes sense to explain the difference between multiprocessing and
```

Comment:
I wasn't really sure where to put this without destroying the flow in the epipsode, so leaving this hanging for now.